reactor-core/src/main/java/reactor/util/retry/Retry.java

```diff
@@ -60,6 +72,16 @@
 	 */
 	public abstract Publisher<?> generateCompanion(Flux<RetrySignal> retrySignals);
 
+	/**
+	 * Return the user provided context that was set at construction time.
+	 *
+	 * @return the user provided context that will be accessible via {@link RetrySignal#retryContextView()}.
+	 */
+	public ContextView retryContext() {
```

Comment:
I'm still unsure about that one. should it be `retryContextView()`? or even simpler, `context()`? (as `Retry` implementors are less likely to implement stuff that is unrelated to retries, unlike eg. `RetrySignal`). In the end, I think I like the simplicity of `retryContext()` though...